app/src/main/res/layout-sw320dp/activity_home_screen.xml

```diff
@@ -56,7 +56,7 @@
         android:layout_height="wrap_content"
         android:onClick="signIn"
         android:text="@string/signInButton"
-        android:backgroundTint="@color/red"
+        android:backgroundTint="@color/primary"
```

Comment:
I hope the new color will be nice !

Reply:
Haha, I don't know, we'll have to choose together what we want but in any case now, everything should be very easy to change!